api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -72,6 +72,36 @@
           }
         }
       }
+    },
+    "successors": {
+      "type": "object",
+      "description": "Buckets usable as destination with \"Push to Bucket\" in web UI.",
```

Comment:
as **a** destination

---

Just noting, I use free Grammarly for this: 
![Screenshot from 2020-12-01 16-48-11](https://user-images.githubusercontent.com/533229/100748954-122ee780-33f5-11eb-8a78-a9e3761b5df0.png)